### PR TITLE
Use latest versions of jquery.form and jquery.tools

### PR DESCRIPTION
**Pull request by msom**

### CODE REVIEW

**Comment by msom** [comment]
Is this ok?

**Comment by jensens** [comment]
looks overall good, but I'm not deep enough into this, requested more opinions.

**Comment by thet** [comment]
Related - already merged - PR: https://github.com/collective/jquerytools/pull/1

File: CHANGES.rst [hunk 1]
```diff
@@ -4,17 +4,13 @@ Changelog
...
-
-New features:
-
-- *add item here*
-
 Bug fixes:
 
-- *add item here*
+- Fix for dotted path form ids.
+  [malsup]
```
Comment (thet) [inline CHANGES.rst]: Please better make a changelog entry like: ``Updated jquery.form to latest 3.51.0``.
Funny side note, looking up jquery.form on Github I found, that a PR reporting & fixing the dotted path form ids bug was done by me. Couldn't remember that.
It's not even included in mockup, which I might take care of later...

File: plone/app/jquerytools/browser/jquery.form.js [hunk 2]
```diff
@@ -10,6 +10,21 @@
  */
 /*global ActiveXObject */
 
+// AMD support
```
Comment (thet) [inline plone/app/jquerytools/browser/jquery.form.js]: Are you sure, you included jquery.form 3.51.0 ? It looks like, the only change here is AMD support.

File: plone/app/jquerytools/browser/jquery.tools.dateinput.js [hunk 3]
```diff
@@ -211,9 +211,8 @@
 		
 		// use sane values for value, min & max		
 		value = parseDate(value) || now;  
-		
-		min   = parseDate(min || new Date(yearNow + conf.yearRange[0], 1, 1));
-		max   = parseDate(max || new Date( yearNow + conf.yearRange[1]+ 1, 1, -1));
+		min   = parseDate(min || new Date(yearNow + conf.yearRange[0], 0, 1));
+		max   = parseDate(max || new Date(yearNow + conf.yearRange[1]+ 1, 0, 0));
```
Comment (thet) [inline plone/app/jquerytools/browser/jquery.tools.dateinput.js]: Looks good - this is the change you made yourself in https://github.com/collective/jquerytools/pull/1/files

File: CHANGES.rst (hunk 1 above)
Comment (thet) [inline CHANGES.rst]: https://github.com/malsup/form/pull/419

**Comment by msom** [comment]
I updated the changelog and ran grunt again (nothing changed)

**Comment by msom** [comment]
Is this now ok?

**Comment by msom** [comment]
@thet @jensens Any chance to merge this and make a new release?

**Comment by thet** [comment]
merged!

for a release you have to ping one of the maintainers of https://pypi.python.org/pypi/plone.app.jquerytools or the @plone/release-team

**Comment by msom** [comment]
Great, thanks!

@mauritsvanrees, can you make a new release?

**Comment by mauritsvanrees** [comment]
I have released this in 1.9.1. Thanks!

**Comment by msom** [comment]
👍🏻 Thanks!

**Pull request merged**